guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -56,6 +56,10 @@ include::topics/upgrading_discovery_satellite.adoc[leveloffset=+3]
 // Configuring Subnets with a Template Capsule
 include::topics/upgrading_discovery_subnet_with_a_template_capsule.adoc[leveloffset=+3]
 
+ifdef::satellite[]
+// Upgrading Satellite From RHEL 7 to RHEL 8
```

Comment:
Maybe Upgrading Foreman? Or just no comment at all because we ideally have a descriptive file name that matches 100% the title.

Reply:
@maximiliankolb No comment at all?